src/main/resources/data/minecraft/tags/blocks/logs.json

```diff
@@ -1,6 +1,6 @@
 {
- "replace": false,
- "values": [
-    "#crimson:logs"
- ]
+	"replace": false,
```

Comment:
"

Reply:
(ditto mark)

Reply:
Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.